### ISSUE TITLE
CPFP calculation if the child has a lower fee rate

### ISSUE THREAD

**Issue by feelancer21** [sparrowwallet/sparrow#1549]
If the child Tx has a lower feer ate than the parent Tx, an effective fee rate is currently calculated that is lower than the fee rate of the parent Tx. 
In my opinion, it makes no sense to calculate an effective Feerate for a CPFP in this case, as the parent Tx can be confirmed independently of the child Tx. 

![image](https://github.com/user-attachments/assets/e60313aa-94e2-4d81-a22c-c2989602ac97)

**Comment by craigraw** [comment]
Agreed. As of 947013e0 the CPFP rate will only be shown if the child's fee rate is higher than the effective fee rate.
